README.md

```diff
@@ -33,6 +33,10 @@ Compared with other graph database solutions, **Nebula Graph** has the following
 
```

Comment:
I prefer that we use clear instructional words to tell people what to do and where to go:
1. This repo hosts the code of versions prior to v1.2.1 and v2.5.0 later
2. If you are looking to use the versions between v2.0.0 and v2.5.0, please head to [nebula-graph](https://github.com/vesoft-inc/nebula-graph)   
3. The data format, rpc protocols, clients, etc. are not compatible between Nebula Graph v1.x and v2.x

Reply:
Thanks, I modify the readme.